### ISSUE TITLE
DataTable cannot be read by Screen Reader tool

### ISSUE THREAD

**Issue by miron4dev** [primefaces/primefaces#2021]
## 1) Environment
- PrimeFaces version: 6.0
- Does it work on the newest PrimeFaces version? No
- Application server + version: JBoss EAP 7.0
- Affected browsers: Internet Explorer 11

## 2) Expected behavior
DataTable can be read by Screen Reader tool.
`table`, `tr`, `td` elements must have ARIA roles like in this example
https://www.paciellogroup.com/blog/2014/10/notes-on-fixing-incorrect-table-structure-using-aria/

## 3) Actual behavior
DataTable cannot be read by Screen Reader tool, e.g. JAWS.

## 4) Steps to reproduce
- Go to http://www.primefaces.org/showcase/ui/data/datatable/scroll.xhtml
- Try to read any table using JAWS

**Comment by miron4dev** [comment]
I can prepare a pull request, if the issue will be accepted.

**Comment by tandraschko** [comment]
Makes sense for me. If you have time, please provide a PR!

**Comment by tandraschko** [comment]
Looks like already implemented: http://localhost:8080/showcase/ui/data/datatable/basic.xhtml
